docs/src/modules/components/MarkdownElement.js

```diff
@@ -21,39 +21,37 @@ const Root = styled('div')(({ theme }) => ({
     padding: theme.spacing(2),
     backgroundColor: blueDark[800],
     colorScheme: 'dark',
-    direction: 'ltr',
+    direction: theme.direction,
     borderRadius: theme.shape.borderRadius,
     border: '1px solid',
     borderColor: blueDark[700],
     overflow: 'auto',
     WebkitOverflowScrolling: 'touch',
-    maxWidth: 'calc(100vw - 32px)',
+    fontSize: theme.typography.pxToRem(13),
     maxHeight: '400px',
+    maxWidth: 'calc(100vw - 32px)',
     [theme.breakpoints.up('md')]: {
       maxWidth: 'calc(100vw - 32px - 16px)',
     },
   },
-  '& code, & code[class*="language-"]': {
-    direction: 'ltr',
+  '& code': {
     display: 'inline-block',
     ...theme.typography.body2,
-    fontSize: theme.typography.pxToRem(13),
     fontFamily: theme.typography.fontFamilyCode,
-    fontWeight: 400,
     WebkitFontSmoothing: 'subpixel-antialiased',
-    padding: '0 5px',
-    borderRadius: 5,
+    fontWeight: 400,
+    // Reset for Safari
+    // https://github.com/necolas/normalize.css/blob/master/normalize.css#L102
+    fontSize: '1em',
```

Comment:
The fix is to not apply the custom font size on the `code` but on the `pre` element.